etl/steps/data/garden/ember/2024-05-08/yearly_electricity.meta.yml

```diff
@@ -2,6 +2,7 @@ definitions:
   common:
     processing_level: major
```

Comment:
This would be a lot of effort, and these indicators are not used in grapher. I have a `description_processing` in the electricity mix (although it doesn't fully explain all processing of each indicator). Doing it properly would only be feasible if we were using a processing log.